docs/guides/clickhouse/dr/overview/index.md

```
@@ -0,0 +1,350 @@
+still hold Keeper quorum keep accepting writes, and the write endpoint follows to a DC
+that holds quorum.
+
+This page is the conceptual overview and a quick start. See also:
+
+- [DC-DR User Guide](/docs/guides/clickhouse/dr/guide/index.md) for every aspect of
+  running in DC-DR mode (components, status, connecting, monitoring, switchover,
+  failback, day-2 ops).
+- [DC-DR Runbook](/docs/guides/clickhouse/dr/runbook/index.md) for what to do in each
+  operational scenario.
+
+> **New to KubeDB?** Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)